### ISSUE TITLE
macOS: error when launching ComfyUI

### ISSUE THREAD

**Issue by tanis2000** [LykosAI/StabilityMatrix#980]
### What happened?

I updated ComfyUI to the latest master as of today from the StabilityMatrix package manager.

When I launch ComfyUI I get the following error:

```
Unobserved Task Exception - IOException
Read-only file system: '/ (0).DS_Store'
```

The file-system is not read-only of course. I have the feeling it might due to some permissions or something like that, but I did not change anything and I haven't updated the macOS version for a long time. This box is still running on Sonoma 14.6.1

### Steps to reproduce

1. Open StabilityMatrix
2. Install the latest master version of ComfyUI
3. Launch ComfyUI

### Version

v11.1

### What Operating System are you using?

macOS

**Comment by tanis2000** [comment]
If I switch the shared models method from symlink to config it does start but then some models are not actually available because SM is overwriting the custom_model_paths.yaml where I wanted to put the other folders with additional models.

**Comment by mohnjiles** [comment]
Might be having trouble moving that .DS_Store file into the shared model directory? Could try deleting that, or check [this comment](https://github.com/LykosAI/StabilityMatrix/issues/882#issuecomment-2347150002) to solve your issue with Config model sharing.

**Comment by tanis2000** [comment]
Yeah I am pretty sure it might not even know what to do with macOS managed files, but it would be good if it dis. And thanks for pointing me in the right direction about the custom config stuff!

**Comment by tanis2000** [comment]
This is still relevant
